spec/active_record/postgres_pub_sub/listener_spec.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "ostruct"
```

Comment:
I haven't tracked this down but we must have had a dependency which was loading this because tests were failing because OpenStruct was not loaded.